src/main/java/org/jabref/gui/util/ThemeLoader.java

```diff
@@ -47,9 +48,10 @@ public ThemeLoader(FileUpdateMonitor fileUpdateMonitor, JabRefPreferences jabRef
 
         String cssVmArgument = System.getProperty("jabref.theme.css");
         String cssPreferences = jabRefPreferences.get(JabRefPreferences.FX_THEME);
+
         if (StringUtil.isNotBlank(cssVmArgument)) {
             // First priority: VM argument
-            LOGGER.info("Using css from VM option: " + cssVmArgument);
+            LOGGER.info("Using css from VM option: {}", cssVmArgument);
```

Comment:
Do you think we still need the VM option? It was meant to ease development, but it should be easy enough now to switch themes / load custom css.